main/src/com/google/refine/exporters/sql/SqlCreateBuilder.java

```diff
@@ -83,11 +84,12 @@ public String getCreateSQL() {
                 
                 if (name != null) {
                     if(trimColNames) {
-                        String trimmedCol = name.replaceAll("[^a-zA-Z0-9_]", "_");
-                        createSB.append( trimmedCol + " ");
-                    }else{
-                        createSB.append(name + " ");
+                        name = name.replaceAll("[^a-zA-Z0-9_]", "_");
+
+                    }if (EscapeCharacter!=null){
```

Comment:
```suggestion
                    }
                    if (EscapeCharacter!=null){
```
This is a visual change, but was misleading the other way.